### ISSUE TITLE
Update Docs on BigQuery fetch_data method

### ISSUE THREAD

**Issue by WillianFuks** [GoogleCloudPlatform/google-cloud-python#3595]
Hello,

After this [issue](https://github.com/GoogleCloudPlatform/google-cloud-python/issues/3576) was posted, docstrings looks ok now. I was wondering if it's possible to update the sphinx [documentation](https://googlecloudplatform.github.io/google-cloud-python/stable/bigquery-query.html#google.cloud.bigquery.query.QueryResults.fetch_data) as well now as it's still showing information regarding the previous return type for the method (and questions about this still pops up in stackoverflow)

Thanks!

**Comment by dhermes** [comment]
@WillianFuks sure thing. The "latest" docs **should** be updated but they aren't because we've been having build issues with flaky APIs (more like flaky tests). You linked to the "stable" docs, which only get updated when we do a release.

**Comment by lukesneeringer** [comment]
This should now be fixed. Please re-open if not.

**Comment by WillianFuks** [comment]
Hi @lukesneeringer , thank you for the update! Just wondering, the link I posted in the question no longer exists. When following through google's [page](https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-python) about the Python API, I ended up in this [docs](https://googlecloudplatform.github.io/google-cloud-python/stable/bigquery/query.html) but it looks like the `fetch_data` is still not updated there. I wonder if I'm in the right docs (or maybe if google's reference needs update?)

Thanks!
